_posts/0000-01-02-MeganDLS.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
![Image of C&H](https://i.insider.com/564cc42e2491f99d008b63b7?width=1136&format=jpeg)